Packages/com.unity.inputsystem/InputSystem/Plugins/UI/MultiplayerEventSystem.cs

```diff
@@ -0,0 +1,16 @@
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+public class MultiplayerEventSystem : EventSystem
```

Comment:
Should this be in our namespace?

Reply:
Yup. `UnityEngine.Experimental.Input.Plugins.UI`.

Reply:
Please add some XML docs.